source/jormungandr/jormungandr/realtime_schedule/tests/sytral_test.py

```diff
@@ -397,8 +402,8 @@ def next_passage_for_route_point_test(mock_good_response):
     """
     sytral = Sytral(id='tata', service_url='http://bob.com/')
 
-    mock_requests = MockRequests({'http://bob.com/?stop_id=42': (mock_good_response, 200)})
-
+    # request http://bob.com/?stop_id=42
```

Comment:
I updated the mocked response with the simple url, because when we use params struct, the *mock.patch* only see simple url and not the params. 
I added a comment for more clarity

Reply:
Maybe you can use this: https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/tests/utils_test.py#L64-L87
Or maybe there are even better tools out there in the wild

Reply:
Hoooo nice. this is what i need :ok_hand: 
Thanks